screens/LoginScreen.js

```diff
@@ -13,10 +48,19 @@ export default function LoginInScreen() {
       imageStyle={styles.backgroundImage}
     >
       <View style={styles.formContainer}>
```

Comment:
If you find yourself using the same colors over and over again, it can be a good idea to store all of your colors as constants in one file.